test/chromium/chromium.spec.ts

```diff
@@ -88,4 +88,18 @@ describe('chromium', (suite, parameters) => {
     // make it work with Edgium.
     expect(serverRequest.headers.intervention).toContain('feature/5718547946799104');
   });
+  it('should support PLAYWRIGHT_CHROMIUM_DEBUG_PORT', async ({requireIsolatedPlaywright, defaultBrowserOptions, parallelIndex}) => {
+    const orig = process.env.PLAYWRIGHT_CHROMIUM_DEBUG_PORT;
+    const port = 9123 + parallelIndex;
+
+    process.env.PLAYWRIGHT_CHROMIUM_DEBUG_PORT = String(port);
+    const playwright = requireIsolatedPlaywright();
```

Comment:
I was already thinking that we should always use the pipe, and additionally pass '--remote-debugging-port' when asked. Now that we need a workaround to test this, let's switch the implementation right away? Also, if we do so, perhaps we don't need the env variable anymore, and let clients just pass "--remote-debugging-port=PORT" themselves? WDYT?

Reply:
We should definitely accept `--remote-debugging-port`. That was the original puppeteer behavior unless someone changed it. I assumed `PLAYWRIGHT_CHROMIUM_DEBUG_PORT` was part of your debugging environment variables story, so I'll  leave the call on whether or not it makes sense to you.